src/public_key.rs

```diff
@@ -385,13 +385,13 @@ impl PublicKey {
 
 #[cfg(feature = "solana")]
 impl TryFrom<PublicKey> for solana_sdk::pubkey::Pubkey {
-    type Error = &'static str;
+    type Error = error::DecodeError;
```

Comment:
nope.. this should be `error::Error` or just `Error` in this case like with all the other TryFrom's there

Reply:
fixed. checking on CI now